src/Uuid/Permalink.php

```diff
@@ -0,0 +1,79 @@
+<?php
+
+namespace Kirby\Uuid;
+
+use Kirby\Cms\App;
+use Kirby\Cms\File;
+use Kirby\Cms\Page;
+use Kirby\Toolkit\Str;
+
+/**
+ * Permalink for \Kirby\Cms\Page or \Kirby\Cms\File
+ *
+ * @package   Kirby Uuid
+ * @author    Nico Hoffmann <nico@getkirby.com>
+ * @link      https://getkirby.com
+ * @copyright Bastian Allgeier
+ * @license   https://getkirby.com/license
+ * @since     6.0.0
+ */
+class Permalink
+{
+	public function __construct(
+		public FileUuid|PageUuid $uuid
+	) {
+	}
+
+	public static function from(string $uuid): static|null
```

Comment:
`Suggestion`: As there are two factory methods that take different input, bringing their names in line could increase the overview:

```php
Permalink::fromUuid(string $uuid)
Permalink::fromUrl(string $url)
```

This would also be consistent with the `uuid()` and `url()` methods that convert in the opposite direction.

Reply:
What would you think about `::for(string $uuid)` and `::from(string $url)`? Kinda similar how the following PR then does it for the main UUID class as well with the uuid/model cases (where here we have uuid/url).

Reply:
Right, that could work!